File: web/src/model/victron_controller/dashboard/from_0_2_0_command_set-mode.ts
// @ts-nocheck
import {SetMode as v0_2_0_SetMode} from './v0_2_0/Command'
import {SetMode as dashboard_SetMode} from './Command'

export function convert__command__set_mode__from__0_2_0(from: v0_2_0_SetMode): dashboard_SetMode {
    return new dashboard_SetMode (
        from.knob_name,
        JSON.parse(JSON.stringify(from.value))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_command_set_mode.rs


pub fn convert__command__set_mode__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::command::SetMode) -> crate::victron_controller::dashboard::command::SetMode {
    crate::victron_controller::dashboard::command::SetMode {
        knob_name: from.knob_name.clone(),
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}